sql/src/main/java/io/crate/executor/transport/executionphases/ExecutionPhasesTask.java

```diff
@@ -168,9 +168,10 @@ public void execute(RowReceiver rowReceiver, Row parameters) {
         try {
             setupContext(operationByServer, handlerPhases, handlerReceivers);
         } catch (Throwable throwable) {
-            return Futures.immediateFailedFuture(throwable);
+            List<ListenableFuture<Long>> failedFuture = new ArrayList<>(1);
+            failedFuture.add(Futures.immediateFailedFuture(throwable));
```

Comment:
is it really okay to only return 1 result instead of nodeOperationTrees.size() failures?
If you, `Collections.singletonList` could be used